### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by Chamepp** [oziw/3D-Web-Roadmap#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * read contributions guidelines
  * Ask questions for more instructions
  * Share your new ideas
  * Participating in Polls
  * Spreading the word
  * Showing your work
  * Answering discussions

  Here you can hear about the repo updates and announcements.

**Comment by sumitshinde-84** [comment]
hii guys  want proper guidence to threejs please help me

**Reply by Chamepp** [reply]
Hi sumit, there is already the threejs path available on the repository, with resources and paths needed
